docs/project_pi4micronaut/about.md

```diff
@@ -16,17 +16,17 @@ The existence of Pi4Micronaut is justified by the need for a robust, scalable, a
 
 - **Source Code:** [https://github.com/oss-slu/Pi4Micronaut/](https://github.com/oss-slu/Pi4Micronaut/) [<img src="/img/git-alt.svg" alt="git" width="25" height="25" />](https://github.com/oss-slu/Pi4Micronaut/)
 - **Client** Jeff Brown - Unity Foundation
-- **Current Tech Lead:** Ruthvik Mannem [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/ruthvikm)
+- **Current Tech Lead:** Yenkatarajalaxmimanohar [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/yrlmanoharreddy)
 - **Developers:**
-  - Adrian Swindle (capstone) [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/SwindleA) 
+  - Ruthvik Mannem (Previous Tech Lead) [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/ruthvikm)
```

Comment:
please also add the "alumni" label to Ruthvik here, and use lowercase "prior tech lead" instead of "Previous Tech Lead" i want to use "prior tech lead"

Reply:
I pushed the changes